index.html

```diff
@@ -40,9 +44,179 @@ <h1 class="description">I am Desmond, glad to see you!
           <li><a href="#"><img src=" ./images/M-icon.svg" alt="M-icon"></a> </li>
           <li><a href="#"><img src=" ./images/hand-raise.svg" alt="handraise icon"></a></li>
         </ul>
-      </section>
-
-      
+      </section> 
     </main>
+    <!-- work section -->
+    <section class="work">
+        <div class="container">
+            <div class="workSection">
+                <!-- Card 1 section -->
+                <div class="card ">
+                    <img class="card-image" src="./images/skills-snapshot.png" alt="skills-snapshot">
+                    <div class="card-content">
+                        <h1 class="card-title">Tonic</h1>
+                        <div class="skills">
+                            <span>CANOPY</span>
```

Comment:
- [ ] For SEO purposes, kindly use an hx tag for the sub-headings

Reply:
- This issue is not yet resolved :x: